dohlistener.go

```diff
@@ -155,6 +155,11 @@ func (s *DoHListener) String() string {
 }
 
 func (s *DoHListener) dohHandler(w http.ResponseWriter, r *http.Request) {
+	if r.URL.Path != "/dns-query" {
```

Comment:
Would it be worth mentioning the path as well as the content-type in the [docs](https://github.com/folbricht/routedns/blob/master/doc/configuration.md#dns-over-https)?

Reply:
I am reconsidering if this should be included. I looked at the RFC but it seems like the path `/dns-query` is not a MUST, rather a norm. 

I can see that this could maybe cause issues, since with routedns users can define different paths than `/dns-query` on the DoH client.

Reply:
yeah, it may be best to avoid breaking existing setups that use a non-standard path. One thing we could do is make the path configurable, and if it's not set, then all paths are allowed. This would not impact existing configs while still allowing someone to enforce a specific path. And it's fits nicely with routedns' overall configurability